packages/logging/charts/logging/charts/logging-operator/crds/logging.banzaicloud.io_flows.yaml

```diff
@@ -0,0 +1,815 @@
+
+---
```

Comment:
actually if these contain only a single object, you can remove the newline and the --- as well

Reply:
Believe these are generated outside of us.